templates/web/cheshireeast/email_sent.html

```diff
@@ -6,17 +6,9 @@ <h1>[% loc("Nearly done! Now check your email&hellip;") %]</h1>
 
```

Comment:
Can drop this entire template now :)

Reply:
Reverted back to what's on master, but can delete if that's preferred.